src/fragment.ts

```diff
@@ -0,0 +1,21 @@
+import { TramOneComponent } from './types';
+import { registerDom } from './dom';
+import useEffect from './use-effect';
+
+const xml = registerDom(null);
```

Comment:
we use `registerDom` here because fragments are universal across HTML and SVG (additionally, the `registerHtml` and `registerSvg` use this component, so we couldn't cyclically depend on it).